provisioning_templates/iPXE/kickstart_default_ipxe.erb

```diff
@@ -9,47 +9,19 @@ oses:
 - RedHat
 %>
 <%
-  rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name != 'Fedora'
-  is_fedora = @host.operatingsystem.name == 'Fedora'
-  os_major = @host.operatingsystem.major.to_i
-  os_minor = @host.operatingsystem.minor.to_i
-  major = @host.operatingsystem.major.to_i
   iface = @host.provision_interface
-  mac = @host.provision_interface.mac
-  hostname = @host.name
-  subnet = @host.subnet
-  options = []
+  subnet4 = iface.subnet
```

Comment:
Nit: these variables are only used once so you might as well use them inline.

Reply:
I am trying here to set some "practice" to always have these variables explicitly set, so when you write `subnet4` it immediately strikes you there's also IPv6. This is intentional, these "headers" often get copied over in this repo.